terraform/monitoring/panels/app/publishing_workers_processing_size.libsonnet

```
@@ -0,0 +1,19 @@
+local grafana   = import '../../grafonnet-lib/grafana.libsonnet';
+local defaults  = import '../../grafonnet-lib/defaults.libsonnet';
+
+local panels    = grafana.panels;
+local targets   = grafana.targets;
+
+{
+  new(ds, vars)::
+    panels.timeseries(
+      title       = 'Messages in processing state',
+      datasource  = ds.prometheus,
+    )
+    .configure(defaults.configuration.timeseries)
+    .addTarget(targets.prometheus(
+      datasource  = ds.prometheus,
+      expr        = 'sum(rate(publishing_queue_processing_size_total{}[$__rate_interval]))',
```

See total, not the rate of total change?

```suggestion
      expr        = 'sum(publishing_queue_processing_size_total{}[$__rate_interval])',
```

---

Thanks for catching this! We don't need the rate here. `_total` and `$__rate_interval` should be removed as well. [example](https://github.com/WalletConnect/echo-server/blob/1905c8bf7e49fcab07453210676843fd531afa41/terraform/monitoring/main.tf#L420).